test/systemtests/init_test.go

```diff
@@ -25,6 +23,7 @@ type systemtestSuite struct {
 }
 type BasicInfo struct {
 	Scheduler    string `json:"scheduler"`      //swarm, k8s or plain docker
+	InstallMode  string `json:"install_mode"`   //legacy or kubeadm
```

Comment:
We might end up use it for swarm-mode as well when the system-tests for that are added. It's still relevant at this point.